core/services/chainlink/relayer_chain_interoperators_test.go

```diff
@@ -0,0 +1,426 @@
+			&solana.SolanaConfig{
+				ChainID: &solanaChainID1,
+				Enabled: ptr(true),
+				Chain:   solcfg.Chain{},
+				Nodes: []*solcfg.Node{{
+					Name: ptr("solana chain 1 node 1"),
+					URL:  ((*relayutils.URL)(models.MustParseURL("http://localhost:8547").URL())),
+				}},
+			},
+			&solana.SolanaConfig{
+				ChainID: &solanaChainID2,
+				Enabled: ptr(true),
+				Chain:   solcfg.Chain{},
+				Nodes: []*solcfg.Node{{
+					Name: ptr("solana chain 2 node 1"),
+					URL:  ((*relayutils.URL)(models.MustParseURL("http://localhost:8527").URL())),
+				}},
+			},
+		}
+
+		c.Starknet = starknet.StarknetConfigs{
+			&starknet.StarknetConfig{
+				ChainID: &starknetChainID1,
+				Enabled: ptr(true),
+				Chain:   stkcfg.Chain{},
+				Nodes: []*stkcfg.Node{
+					{
+						Name: ptr("starknet chain 1 node 1"),
+						URL:  ((*relayutils.URL)(models.MustParseURL("http://localhost:8547").URL())),
+					},
+					{
+						Name: ptr("starknet chain 1 node 2"),
+						URL:  ((*relayutils.URL)(models.MustParseURL("http://localhost:8548").URL())),
+					},
+					{
+						Name: ptr("starknet chain 1 node 3"),
+						URL:  ((*relayutils.URL)(models.MustParseURL("http://localhost:8549").URL())),
+					},
+				},
+			},
+			&starknet.StarknetConfig{
+				ChainID: &starknetChainID2,
+				Enabled: ptr(true),
+				Chain:   stkcfg.Chain{},
+				Nodes: []*stkcfg.Node{
+					{
+						Name: ptr("starknet chain 2 node 1"),
+						URL:  ((*relayutils.URL)(models.MustParseURL("http://localhost:3547").URL())),
+					},
+				},
+			},
+		}
+
+		c.Cosmos = cosmos.CosmosConfigs{
+			&cosmos.CosmosConfig{
+				ChainID: &cosmosChainID1,
+				Enabled: ptr(true),
+				Chain: coscfg.Chain{
+					GasLimitMultiplier: ptr(decimal.RequireFromString("1.55555")),
+				},
+				Nodes: cosmos.CosmosNodes{
+					&coscfg.Node{
+						Name:          ptr("cosmos chain 1 node 1"),
+						TendermintURL: (*relayutils.URL)(models.MustParseURL("http://localhost:9548").URL()),
+					},
+				},
+			},
+			&cosmos.CosmosConfig{
+				ChainID: &cosmosChainID2,
+				Enabled: ptr(true),
+				Chain: coscfg.Chain{
+					GasLimitMultiplier: ptr(decimal.RequireFromString("0.777")),
+				},
+				Nodes: cosmos.CosmosNodes{
+					&coscfg.Node{
+						Name:          ptr("cosmos chain 2 node 1"),
+						TendermintURL: (*relayutils.URL)(models.MustParseURL("http://localhost:9598").URL()),
+					},
+				},
+			},
+		}
+	})
+
+	db := pgtest.NewSqlxDB(t)
+	keyStore := cltest.NewKeyStore(t, db, cfg.Database())
+
+	lggr := logger.TestLogger(t)
+
+	factory := chainlink.RelayerFactory{
+		Logger:       lggr,
+		DB:           db,
+		QConfig:      cfg.Database(),
+		LoopRegistry: plugins.NewLoopRegistry(lggr),
+		GRPCOpts:     loop.GRPCOpts{},
+	}
+
+	testctx := testutils.Context(t)
+
+	tests := []struct {
+		name      string
+		initFuncs []chainlink.CoreRelayerChainInitFunc
+
+		expectedEVMChainCnt   int
+		expectedEVMNodeCnt    int
+		expectedEVMRelayerIds []relay.Identifier
+
+		expectedSolanaChainCnt   int
```

Comment:
/nit imho no shorthand > shorthand except in extreme cases. Here, I think readers don't gain much from shortening `Count` to `Cnt`